### ISSUE TITLE
refactor: rename import_alias -> namespace_alias

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1138]

**Comment by SamInverted** [comment]
Does this mean anything named with import_alias should be renamed to namespace_alias?

**Comment by strager** [comment]
Anything related to the following syntax is a namespace alias (despite the fact that the keyword `import` is used):
```typescript
import ns = otherns;
```
Anything related to the following syntax is an import alias (thus shouldn't be renamed):
```typescript
import m = require('mod');
```
(I forgot about the second case when I filed this issue.)

This task would involve going through each instance of `import_alias` or `import alias` in the code, figuring out whether it's an import alias or a namespace alias, and renaming when it's a namespace aliases.

**Comment by cmstoddard** [comment]
@strager close this?

**Comment by strager** [comment]
@cmstoddard No, this issue is still present.

**Comment by UnfairBots** [comment]
Hello, I have a few questions:
- I found some files in test folder, which has some function name like ...import_alias... but verifies namespace alias, does it mean to change function name to ...namespace_alias...?
- In both cases import alias and namespace alias you mentioned, the same error message display (TypeScript import aliases are not allowed in JavaScript- [E0274](https://quick-lint-js.com/errors/E0274/)), do we also need to have a new message to match with namespace alias case? If yes that would be a new implementation, not refactor.

**Comment by strager** [comment]
> I found some files in test folder, which has some function name like ...import_alias... but verifies namespace alias, does it mean to change function name to ...namespace_alias...?

Yes.

> In both cases import alias and namespace alias you mentioned, the same error message display (TypeScript import aliases are not allowed in JavaScript- [E0274](https://quick-lint-js.com/errors/E0274/)), do we also need to have a new message to match with namespace alias case?

I wasn't thinking about this, but it'd be nice to have different messages for the different cases, yes.
